### PR TITLE
ztp: Reduce kubelet CPU usage

### PR DESCRIPTION
**Pull request by imiller0**
Increase the housekeeping and eviction monitoring intervals. The CPU use
of kubelet increases (approximately) linearly with increasing numbers of
pods driven largely by these tasks. Increasing the intervals brings CPU
usage back closer to baseline.

Signed-off-by: Ian Miller <imiller@redhat.com>

### CODE REVIEW

File: feature-configs/deploy/container-mount-namespace/container-private-mounts/mc-container-mount-namespace.yaml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
 metadata:
-  name: container-mount-namespace
+  name: container-mount-namespace-and-kubelet-conf
```
Comment (lack) [inline feature-configs/deploy/container-mount-namespace/container-private-mounts/mc-container-mount-namespace.yaml:4]: I'm actually surprised that the name change at this location didn't break the CI lanes.  We could just leave this file unchanged, but it's okay as long as it didn't break cnf-tests.

File: ztp/source-crs/extra-manifest/01-container-mount-ns-and-kubelet-conf.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
 metadata:
-  name: container-mount-namespace
+  name: container-mount-namespace-and-kubelet-conf
```
Comment (lack) [inline ztp/source-crs/extra-manifest/01-container-mount-ns-and-kubelet-conf.yaml:4]: Please remove this file and replace it with a symlink to ../MachineConfigContainerMountNS.yaml

**Review by lack — CHANGES_REQUESTED** [review]

File: feature-configs/deploy/container-mount-namespace/container-private-mounts/mc-container-mount-namespace.yaml (hunk 1 above)
Comment (imiller0) [reply feature-configs/deploy/container-mount-namespace/container-private-mounts/mc-container-mount-namespace.yaml:4]: I thought the same thing. The CI lane looks for a machine config with the container-mount-namespace unit in it. The specific name of the MC isn't checked.

File: ztp/source-crs/extra-manifest/01-container-mount-ns-and-kubelet-conf.yaml (hunk 2 above)
Comment (imiller0) [reply ztp/source-crs/extra-manifest/01-container-mount-ns-and-kubelet-conf.yaml]: Done

**Pull request merged**